src/util/path.js

```diff
@@ -1,17 +1,30 @@
 (function() {
-
   var _join = Array.prototype.join;
 
+  function segmentToBezier(th2, th3, cosTh, sinTh, rx, ry, cx1, cy1, mT, fromX, fromY) {
+    var costh2 = fabric.util.cos(th2),
```

Comment:
Hi, just for sure, this PR means the fabric.js will support Bezier Curve soon?

Reply:
What do you mean? fabric always drawn beizer curves but does not give you the ability to interact and modify them.

Reply:
Oh yes, I mean if this PR could give us ability to modify shapes with Becker Curves.

Reply:
This PR aims at removing everything that is not M L C Q from complex path. so that is easier to write code to handle paths manually.
So yes, it will help definitely with you text/path thing.